pkgs/development/tools/conftest/default.nix

```diff
@@ -13,16 +13,24 @@ buildGoModule rec {
 
   vendorSha256 = "sha256-jI5bX6S2C0ckiiieVlaRNEsLS/5gGkC3o/xauDtCOjA=";
 
-  doCheck = false;
-
   ldflags = [
     "-s"
     "-w"
     "-X github.com/open-policy-agent/conftest/internal/commands.version=${version}"
   ];
 
+  HOME = "$TMPDIR";
```

Comment:
It's _usually_ better to set this just in the phase which is using it in case it interferes with the build process.

Reply:
Have you been able to make it work that way? If yes, let me know what I'm getting wrong.